File: python/7 kyu/94.py
# https://www.codewars.com/kata/5a092d9e46d843b9db000064/train/python

# Passed

def solve(arr):
    all_numbers = set(arr)
    for num in all_numbers:
        if -num not in all_numbers:
            return num
        
output = solve([-110, 110, -38, -38, -62, 62, -38, -38, -38])
print(output)

File: nodejs/7 kyu/94.js
// https://www.codewars.com/kata/57ef016a7b45ef647a00002d/train/javascript

// Passed

function filterHomogenous(arrays) {
  const homogeneousArrays = arrays.filter(
    (subArr) =>
      (subArr.length > 0) &
      subArr.every((elem) => typeof elem === typeof subArr[0])
  );
  return homogeneousArrays;f
}

const output = filterHomogenous([
  [123, 234, 432],
  ["", "abc"],
  [""],
  ["", 1],
  ["", "1"],
  [],
]);
console.log(output);